products/gerrit.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Gerrit Code Review
+category: server-app
+iconSlug: "gerrit"
+permalink: /gerrit
+versionCommand: "java -jar gerrit.war version"
+releasePolicyLink: https://www.gerritcodereview.com/support.html
+changelogTemplate: https://www.gerritcodereview.com/__RELEASE_CYCLE__.html#{{"__LATEST__"| replace:'.',''}}
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:docker/gerritcodereview/gerrit
+
+auto:
+-   git: https://github.com/GerritCodeReview/gerrit.git
+    regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)(\.(?<tiny>[0-9]+))?$
+
+releases:
```

Comment:
Since we only have 1 axis of support here (eol = no security updates), suggestion:

1. Set `activeSupportColumn: false`
2. Drop any `support: ` fields
3. Use the dates from `support` in `eol` instead. Or set it as boolean accordingly.

Reply:
done